ProcessMaker/Policies/ProcessRequestTokenPolicy.php

```diff
@@ -55,7 +57,8 @@ public function update(User $user, ProcessRequestToken $processRequestToken)
     {
         if (
             $processRequestToken->user_id === $user->id ||
-            $processRequestToken->user_id === app(AnonymousUser::class)->id
+            $processRequestToken->user_id === app(AnonymousUser::class)->id ||
+            $processRequestToken->process()->first()?->manager_id === $user->id
```

Comment:
`$processRequestToken->process()->first()` always executes the query to get the process row, maybe you could consider to use `$processRequestToken->process`

Reply:
First observation fixed. POs recommend to to use the current Process Manager, so I fixed using directly ->process
https://processmaker.atlassian.net/browse/FOUR-6660?focusedCommentId=292519